test/helpers/TestHelper.sol

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "forge-std/Test.sol";
+
+import "src/LBFactory.sol";
+import "src/LBPair.sol";
+import "src/LBRouter.sol";
+import "src/LBQuoter.sol";
+import "src/LBErrors.sol";
+import "src/interfaces/ILBRouter.sol";
+import "src/interfaces/IJoeRouter02.sol";
+import "src/LBToken.sol";
+import "src/libraries/Math512Bits.sol";
+import "src/libraries/Constants.sol";
+
+import "test/mocks/WAVAX.sol";
+import "test/mocks/ERC20.sol";
+import "test/mocks/FlashloanBorrower.sol";
+import "test/mocks/ERC20TransferTax.sol";
+
+abstract contract TestHelper is Test, IERC165 {
+    using Math512Bits for uint256;
+
+    uint24 internal constant ID_ONE = 2 ** 23;
```

Comment:
I like `1 << 23` more, but both works. It's up to you

Reply:
In tests I think I prefer writing powers of 2, I find it slightly more readable